docs/devsource/conf.py

```diff
@@ -65,7 +67,7 @@
 
 #autosummary_generate = True
 autosectionlabel_prefix_document = True
-autodoc_mock_imports = ['numba', 
+'''autodoc_mock_imports = ['numba', 
```

Comment:
Is it commented out and the code is not needed any more? If it is not needed, maybe it is better to delete it?

Reply:
Actually I left it commented because it prevented all those import errors I was getting from autodoc before building hpat. It had taken some time to create this list of mock imports and in future if I get these errors again it will be easier to debug with this list. But now that I think of I'll just remove this code and save this list somewhere else. We don't want to clutter up the actual file with all this.